src/main/java/com/alkemy/ong/controller/TestimonialController.java

```diff
@@ -32,4 +34,10 @@
         this.testimonialService.deleteTestimonial(id);
         return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
     }
+    //Pagination, role USER
+    @GetMapping
+    public ResponseEntity<?> getPage(@RequestParam Integer page) {
```

Comment:
we can use an int variable instead? faster, what do you think?

Reply:
Done!